content/shmem_alltoall.tex

```diff
@@ -100,12 +100,16 @@
     If \VAR{team} compares equal to \LibConstRef{SHMEM\_TEAM\_INVALID} or is
     otherwise invalid, the behavior is undefined.
 
-    Before any \ac{PE} calls a \FUNC{shmem\_alltoall} routine,
-    the following conditions must be ensured:
+    Before any \ac{PE} calls a \FUNC{shmem\_alltoall} routine, the following
+    conditions must be ensured, otherwise the behavior is undefined:
     \begin{itemize}
-    	\item The \VAR{dest} data object on all \acp{PE} in the team is
-    	ready to accept the \FUNC{shmem\_alltoall} data.
+        \item The \dest{} array on all \acp{PE} in the team is ready to
+            accept the result of the operation.
+        \item The \source{} buffer at the local \ac{PE} is ready to be
+            read by any \ac{PE} in the team.
```

Comment:
Is there a particular reason for interchanging "buffer" and "array"? (dest array vs. source buffer)?

In the arguments description both are "Symmetric address of an array"

Reply:
Good catch, I will change this to "array" throughout.

Reply:
done